src/containers/NetworkPanel/NetworkDetails/RequestView.tsx

```diff
@@ -112,18 +130,41 @@ const SingleRequestView = (props: SingleRequestViewProps) => {
 }
 
 type RequestViewSectionProps = {
+  id: RequestViewSectionId
   title: string
 }
 
 const RequestViewSection: FC<RequestViewSectionProps> = (props) => {
-  const { title, children } = props
+  const { id, title, children } = props
+  const { collapsedSections, setIsSectionCollapsed } = useRequestViewSections()
+
+  function handleToggleView() {
```

Comment:
Not sure about the convention, I would go for arrow function variable as there are plenty in the project already.

Reply:
Fixed.

Reply:
Can you make the change for the other functions?